### DISCUSSION TITLE
Using Sweetalert2

### DISCUSSION THREAD

**Discussion by Denysetya1** [filamentphp/filament#9986] [Help] [answered]
Hi, sorry I was wondering if there is a way to use sweetalert2 to be a success notification in saving, changing, or deleting data?

**Comment by leandrocfe** [comment, marked as answer]
Maybe this? https://github.com/jantinnerezo/livewire-alert

If you need to register assets, check this section
https://filamentphp.com/docs/3.x/support/assets

If you need to inject something, check this section
https://filamentphp.com/docs/3.x/support/render-hooks
